egs2/ipapack/asr1/README.md

```diff
@@ -0,0 +1,195 @@
+    * Creative Commons license
+    * [1] put the languages with a Creative Commons Non-Commercial license in the test set.
+    * As required, we cite each constituent dataset within DoReCo.
+* MSWC (Multilingual Spoken Word Corpus) [4] - clips of single words padded to 1 second derived with forced alignment from Mozilla Common Voice across 50 languages (36 in IPAPack)
+    * Creative Commons license
+
+As such, this recipe contains three test sets: `test_fleurs` (162 hours), `test_doreco` (5 hours), and `test_mswc` (64 hours).
+Regarding the train/dev/test splits more broadly, we followed [1]'s original train/dev/test splits exactly.
+In the test sest, there are at least five unseen languages: Vietnamese, Tamil, Hausa, Georgian, and Odia.
+
+The dataset uses IPA for phonemic transcriptions (as opposed to ARPAbet) and did not perform normalization (see "Phoneme tokenizer" in [1]). However, all phonemes in the training data are supported by panphon / epitran.
+
+Note: a 20,000 hour will be released soon, with consistent and clean Unicode tokenization of IPA symbols.
+
+
+In the results below, we can interpret the WER as phoneme error rate, because each phoneme is its own word after preprocessing the transcriptions.
+
+
+* GPU: 4 V100-32GB
+* Because the dataset is large, it is recommended to use an additional amount of CPU.
+* The results below only report those on test_doreco due to resource constraints.
+* `asr_train_asr_transformer_raw_word` is a simple encoder-decoder trained from scratch.
+* `asr_train_asr_xeus_transformer_extracted_word` is an encoder-decoder using XEUS features extracted in an offline fashion (meaning XEUS is not finetuned).
+    * ```pip install 'espnet @ git+https://github.com/wanchichen/espnet. git@ssl'
+    git lfs install
+    git clone https://huggingface.co/espnet/XEUS```
+    https://www.wavlab.org/activities/2024/xeus/
+    * `./local/dump_xeus_feature.sh` to extract XEUS features offline.
+        * This should be run after Stages 3 and 4.
+
+
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Mon Nov 25 23:30:12 EST 2024`
+- python version: `3.9.20 | packaged by conda-forge | (main, Sep 22 2024, 14:11:32)  [GCC 13.3.0]`
+- espnet version: `espnet 202402`
+- pytorch version: `pytorch 2.3.0+cu121`
+- Git hash: `6eadc31daa4e2bd8b3f02bab8c57dd4d539d9f23`
+  - Commit date: `Sun Nov 24 23:12:44 2024 -0500`
+
+## exp/asr_train_asr_transformer_raw_word
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_asr_model_valid.acc.best/test_doreco|5093|194760|53.9|34.4|11.8|4.7|50.9|100.0|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_asr_model_valid.acc.best/test_doreco|5093|396745|70.3|17.5|12.2|3.3|33.0|100.0|
+
+### TER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
```

Comment:
We can simply remove TER if not used (same for WER if it is not properly used here)